src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-assessment-dashboard.component.html

```diff
@@ -0,0 +1,181 @@
+<div *ngIf="exercise">
+    <h4 [translateValues]="{ exerciseTitle: exercise.title }" jhiTranslate="artemisApp.programmingAssessment.dashboard.heading"></h4>
+    <jhi-assessment-filters (filterChange)="updateFilteredSubmissions($event)" [submissions]="submissions"></jhi-assessment-filters>
+    <ng-container *ngIf="!busy"></ng-container>
```

Comment:
There is nothing inside this ng-container, am I missing something?

Reply:
This part I just copied from the other _exercisetype_-assessment-dashboards. As it turns out in a previous version of this code (in the text-assessment-dashboard e.g.) there was an else branch if the condition was true. I will delete it.